src/compass.js

```diff
@@ -46,6 +47,9 @@ function clearCompass() {
 
 function draw(what) {
   console.log("Call: Compass.draw");
+  compass.backgroundColor(Themes.currentTheme().textColour);
```

Comment:
Could be moved into a separate update method but only in init means that changing teams from the Pebble app with the watch app open does nothing. Updating it in draw ensure that drawing happens in the correct colour each time. Could have init check for previous init calls, make another method or use draw?